apps/omg/lib/omg/state/transaction/fee.ex

```diff
@@ -0,0 +1,114 @@
+# Copyright 2019 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.State.Transaction.Fee do
+  @moduledoc """
+  Internal representation of a fee claiming transaction in plasma chain.
+  """
+  alias OMG.Crypto
+  alias OMG.Output
+  alias OMG.State.Transaction
+
+  @fee_output_type_marker <<0xFE, 0xE0>>
```

Comment:
reference the map in config somehow or place a TODO to figure out how to nicen this bimapping (as in other occurrences of this I suppose, i.e. `Payment`)

Reply:
I'm adding TODO because it's not obvious, e.g. for `FungibleToken` many markers can point to it.